mycpp/const_pass.py

```diff
@@ -123,6 +164,23 @@ def oils_visit_log_call(self, fmt: StrExpr,
         for i, arg in enumerate(args):
             self.accept(arg)
 
+    def oils_visit_class_def(
+            self, o: ClassDef, base_class_sym: Optional[util.SymbolPath],
+            current_class_name: Optional[util.SymbolPath]) -> None:
+
+        for stmt in o.defs.body:
+            if isinstance(stmt, FuncDef):
+                self.method_defs.Add(current_class_name, stmt.name)
+                self.class_namespaces.Set(current_class_name,
+                                          self.current_file_name)
+        super().oils_visit_class_def(o, base_class_sym, current_class_name)
+
+    def oils_visit_mypy_file(self, o: MypyFile) -> None:
+        if util.ShouldSkipPyFile(o):
```

Comment:
I think we should not need this ShouldSkip()?

In visitor, `visit_mypy_file()` calls it, and then it doesn't calls `oils_visit_mypy()` if it's true

The `oils_visit_*` methods are supposed to already be "narrowed"